akka-persistence-typed-tests/src/test/scala/docs/akka/persistence/typed/ReplicatedAuctionExampleSpec.scala

```diff
@@ -199,7 +199,7 @@ object ReplicatedAuctionExampleSpec {
             case Close =>
               context.log.info("Close")
               require(shouldClose(state))
-              // TODO send email (before or after persisting)
+              // send email (before or after persisting)
```

Comment:
But I think that the whole comment can be removed. No email is being sent (as far I can tell).